oozie/test_oozie.py

```diff
@@ -41,5 +46,57 @@ def test_oozie(self, configuration, machine_suffixes):
             self.verify_instance("{}-{}".format(self.getClusterName(),
                                                 machine_suffix))
 
+    def __submit_pyspark_job(self, configuration, job_args):
+        if self.getImageOs() == 'rocky':
+            self.skipTest("Not supported in Rocky Linux-based images")
+
+        if self.getImageVersion() >= pkg_resources.parse_version("2.1"):
+            python_version = self.PYTHON3_VERSION
+        else:
+            python_version = self.PYTHON2_VERSION
+
+        self.createCluster(
+            configuration,
+            self.INIT_ACTIONS,
+            machine_type="e2-standard-4",
+            boot_disk_size="200GB"
+        )
+        self.assert_dataproc_job(self.getClusterName(), 'pyspark',
+                                 '{}/{}/{} --properties=spark.pyspark.python={},spark.pyspark.driver.python={}'
+                                 ' -- {}'
+                                 .format(self.INIT_ACTIONS_REPO,
+                                         self.COMPONENT,
+                                         self.TEST_SCRIPT_JOB,
+                                         python_version,
+                                         python_version,
+                                         job_args))
+
+    def test_distcp_job(self):
+        self.__submit_pyspark_job(self.CONFIGURATION, "distcp")
+
+    def test_hive_job(self):
```

Comment:
We should also add a hive2 test. Do you want to do this as part of this or do you have a separate plan for new tests?

Reply:
The test added for hive is passing "hive2" as an argument. So, I guess this would suffice the condition. I can take this on as a separate enhancement for this test.

Reply:
Adding functionality for hive is being taken as a separate task and will be considered as an enchancement.